### ISSUE TITLE
Reverse traceroutes not collected or shown

### ISSUE THREAD

**Issue by candlerb** [perfsonar/toolkit#79]
I am running perfsonar 3.5 with automatic patching enabled.

Given a mesh configuration which includes this:

```
<test_spec traceroute_test>
  type              traceroute           # Perform a traceroute test
  test_interval     600                  # Run the test every 10 minutes
  timeout           30                   # Wait 30 seconds
  waittime          5                    # Number of seconds to wait for a response before assuming its lost
  force_bidirectional 1        # <<< NOTE
</test_spec>
```

I was expecting:
- force_bidirectional to cause regular_testing to use bwtraceroute to perform reverse traceroutes
- those reverse traceroutes to be added into the esmond archive
- those traceroutes to be selectable via the psTracerouteViewer

What I find is: the `/toolkit/` front page where it lists the hosts has a `traceroute` link, which goes to psTracerouteViewer. This shows the `A ----> B` test, but there are no `B ----> A` results. The drop-down selection has no `B ----> A` result., only `A ----> other` selections.

I can see from browsing the esmond tree directly at http://x.x.x.x/esmond/perfsonar/archive/?format=json that there are archives created for both `A->B` and `B->A` traceroutes.  However, manually browsing to one of the `B->A` archives I find it is empty. That is:

http://x.x.x.x/esmond/perfsonar/archive/<id/packet-trace/base?format=json> returns only `[]`

I have checked that at the command line, `bwtraceroute -s <remotehost>` is able to do a traceroute from the remote host back to this one.

**Comment by bltierney** [comment]
Andy: has this been addressed?

**Comment by arlake228** [comment]
Fixed in 4.0
